constants/rewards.ts

```diff
@@ -47,22 +47,38 @@ export const WEEKLY_UMA_REWARDS: { [key: string]: any[] } = {
       endDate: Date.UTC(2020, 9, 1, 0, 0, 0),
     },
   ], // uUSDrBTC-DEC
+  "0x90f802c7e8fb5d40b0de583e34c065a3bd2020d8": [
+    {
+      token: "UMA",
+      count: 15000, // This is an approximation based on expected dev mining allocation
+      getPrice: getUmaPrice,
+      startDate: Date.UTC(2020, 12, 25, 23, 0, 0, 0),
+    },
+  ], // YD-ETH-MAR21
+  "0x002f0b1a71c5730cf2f4da1970a889207bdb6d0d": [
+    {
+      token: "UMA",
+      count: 10000, // This is an approximation based on expected dev mining allocation
```

Comment:
Since LM mining estimation takes into account dev mining, this value does not need to be hardcoded, but it doesnt harm anything leaving it in. The estimation does assume 50% of dev mining goes to LM